lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -114,7 +115,9 @@ sub _hmt_csv {
     my $hmt_ref = {};
 
     my $csv = Text::CSV->new({binary => 1}) or die "Cannot use CSV: " . Text::CSV->error_diag();
-    open $fh, '+>', undef or die "Could not open anonymous temp file - $!";                    ## no critic (RequireBriefOpen)
+    #open $fh, '+>', undef or die "Could not open anonymous temp file - $!";                       ## no critic (RequireBriefOpen)
+    my $temp = Path::Tiny->tempfile or die "Could not open anonymous temp file - $!";
```

Comment:
we needn't create a tempfile and print into it and read it back again here:
```suggestion
    open(my $fh, "<", \$content);
```

Reply:
remove old code, updated as suggested